<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>waterMarkWindow</class>
 <widget class="QMainWindow" name="waterMarkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加水印</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>131</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>471</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>水印设置</string>
    </property>
    <widget class="QRadioButton" name="rbtn_addText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>添加水印文字</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>水印文字：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ledit_text">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_fontSet">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>字体设置</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtn_addImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>添加图片水印</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_brower">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ledit_addImage">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>水印图片</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_loadImage">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>加载图片</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>471</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>透明度以及位置设置</string>
    </property>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>透明度：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>水印位置：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>路径设置</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>保存位置：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ledit_savePostion">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_brower2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_execute">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>420</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>执行</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>665</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
